dev-python/mkdocs-bootstrap/mkdocs-bootstrap-1.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Bootstrap theme for MkDocs"
+HOMEPAGE="https://www.mkdocs.org https://github.com/mkdocs/mkdocs-bootstrap"
+SRC_URI="https://github.com/mkdocs/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="doc"
+
+DEPEND="dev-python/mkdocs"
+
+python_compile_all() {
+	default
+	if use doc; then
+		mkdocs build || die "failed to make docs"
+		HTML_DOCS="site"
```

Comment:
PMS states that a global variable shouldn't be modified inside phases, but I don't mind about this one. However it should be an array, because it now gets converted to an array before `einstalldocs` calls.